Misc doc changes

kianmeng opened the pull request:
Besides other documentation changes, this commit ensures the generated
HTML doc for HexDocs.pm will become the source of truth for this Elixir
library and leverage on latest features of ExDoc.

hertogp commented:
Hi Kian,
Thanks for the interest and better documentation principles, e.g. by using the assets subdir.

However, removing the empty `iex>` from the doctests actually breaks them and now `mix test` fails.
If you add those back in & the workflow check succeeds, I would happily merge your pull request.

Thanks,
Pieter

ps. I am new to Elixir, so your help is really appreciated.

kianmeng committed:
Misc doc changes

Besides other documentation changes, this commit ensures the generated
HTML doc for HexDocs.pm will become the source of truth for this Elixir
library and leverage on latest features of ExDoc.

kianmeng commented:
@hertogp I see. Thanks for letting me know. Changes updated.

hertogp commented:
Thanks!
